deepchecks/vision/datasets/classification/mnist.py

```diff
@@ -0,0 +1,60 @@
+import os.path
+
+from torch import nn
+import torch.nn.functional as F
+import torch
+import torchvision
+
+
+import pathlib
+current_path = pathlib.Path(__file__).parent.resolve()
+model_path = str(current_path).replace('\\', '/') + '/models/mnist.pth'
+
+
+class MNistNet(nn.Module):
```

Comment:
This file doesn't really work due to model_path. Needs refactor. Suggest removing the model for now and re-adding it once it works standalone.